packages/sdk-codegen-scripts/src/declarationMiner.ts

```diff
@@ -127,8 +128,11 @@ export class DeclarationMiner {
   }
 
   get remoteOrigin(): string {
+    if (this.originOverride) {
+      return this.originOverride
+    }
     process.chdir(this.sourcePath)
-    return getRemoteHttpOrigin()
+    return getRemoteOrigin()
```

Comment:
Is this what we really want. I looked at the code and it executes
`git remote get-url origin`
which returns
`git@github.com:looker-open-source/sdk-codegen.git`
Looke like getRemoteHttpOrigin attemps to convert it

Reply:
Yes. Note this is declaration miner, not example miner. The address for the helltool code repo is no longer https when you ask for the remote origin. That's why I added the `origin_override` configuration and alternate path composition.